README.md

```diff
@@ -1,3 +1,156 @@
 # Spec Math
+## Overview
+Spec Math is a platform for performing operations on OpenAPI specifications. 
+[OpenAPI](https://github.com/OAI/OpenAPI-Specification) is the industry-standard
```

Comment:
"is the" might be seen as overlooking other standard formats (e.g. RAML) so "is an" might be better here.